api/tests/integration/domain/usecases/start-writing-campaign-profiles-collection-results-to-stream_test.js

```diff
@@ -307,6 +307,7 @@ describe('Integration | Domain | Use Cases | start-writing-profiles-collection-c
           `"'${campaign.name}";` +
           `"'${schoolingRegistration.lastName}";` +
           `"'${schoolingRegistration.firstName}";` +
+          `"${schoolingRegistration.group}";` +
           `"${schoolingRegistration.studentNumber}";` +
           `"'${campaignParticipation.participantExternalId}";` +
           '"Oui";' +
```

Comment:
**suggestion** : changer le nom du test `should return the complete line with student number` en `should return the complete line with student number and group`

Reply:
Suite au [commentaire](https://github.com/1024pix/pix/pull/3553/commits/686efe5ecb4c82b5479b0217a3732432f1ab3d1f#r721201890) d'Arthur, je suggère que nous le nommions de la manière suivante : `should return the complete line for a SUP organisation that manages students`